src/sage/graphs/generators/smallgraphs.py

```diff
@@ -456,7 +456,7 @@ def Cell600(embedding=1):
     EXAMPLES::
 
         sage: # long time
-        sage: g = graphs.Cell600()
+        sage: g = graphs.Cell600()                                                      # needs sage.rings.number_field
```

Comment:
Remaining doctest warnings in `src/sage/graphs/generators/smallgraphs.py`, introduced in this PR:
```
sage -t --long --warn-long 34.0 --random-seed=75874225553897883545888612139197852719 src/sage/graphs/generators/smallgraphs.py
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 460, in sage.graphs.generators.smallgraphs.Cell600
Warning: Variable 'g' referenced here was set only in doctest marked '# long time, needs sage.rings.number_field'
    g.size()
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 462, in sage.graphs.generators.smallgraphs.Cell600
Warning: Variable 'g' referenced here was set only in doctest marked '# long time, needs sage.rings.number_field'
    g.is_regular(12)
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 464, in sage.graphs.generators.smallgraphs.Cell600
Warning: Variable 'g' referenced here was set only in doctest marked '# long time, needs sage.rings.number_field'
    g.is_vertex_transitive()
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 536, in sage.graphs.generators.smallgraphs.Cell120
Warning: Variable 'g' referenced here was set only in doctest marked '# long time, needs sage.rings.number_field'
    g.size()
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 538, in sage.graphs.generators.smallgraphs.Cell120
Warning: Variable 'g' referenced here was set only in doctest marked '# long time, needs sage.rings.number_field'
    g.is_regular(4)
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 540, in sage.graphs.generators.smallgraphs.Cell120
Warning: Variable 'g' referenced here was set only in doctest marked '# long time, needs sage.rings.number_field'
    g.is_vertex_transitive()
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 4759, in sage.graphs.generators.smallgraphs.MathonStronglyRegularGraph
Warning: Variable 'G' referenced here was set only in doctest marked '# long time, needs sage.libs.gap'
    G.is_strongly_regular(parameters=True)   # long time
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 4766, in sage.graphs.generators.smallgraphs.MathonStronglyRegularGraph
Warning: Variable 'G' referenced here was set only in doctest marked '# long time, needs sage.libs.gap'
    G.is_strongly_regular(parameters=True)
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 4769, in sage.graphs.generators.smallgraphs.MathonStronglyRegularGraph
Warning: Variable 'G' referenced here was set only in doctest marked '# long time, needs sage.libs.gap'
    G.is_strongly_regular(parameters=True)
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 4793, in sage.graphs.generators.smallgraphs.JankoKharaghaniGraph
Warning: Variable 'g' referenced here was set only in doctest marked '# long time, needs sage.libs.pari'
    g.is_strongly_regular(parameters=True)  # long time
**********************************************************************
File "src/sage/graphs/generators/smallgraphs.py", line 4885, in sage.graphs.generators.smallgraphs.JankoKharaghaniTonchevGraph
Warning: Variable 'Gamma' referenced here was set only in doctest marked '# long time, needs sage.libs.gap'
    Gamma.is_strongly_regular(parameters=True)  # long time
    [574 tests, 12.13 s]
```

Reply:
Thanks! Fixed in e110826f58d